src/components/Tiptap.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { EditorContent, useEditor } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import TaskItem from '@tiptap/extension-task-item';
+import TaskList from '@tiptap/extension-task-list';
+import Highlight from '@tiptap/extension-highlight';
+import TextAlign from '@tiptap/extension-text-align';
+import MenuBar from './tiptap/MenuBar';
+
+const TipTap: React.FC = () => {
+  const editor = useEditor({
+    extensions: [
+      StarterKit.configure({
+       
+      }),
+      TextAlign.configure({
+        types: ['heading', 'paragraph'],
+      }),
+      TaskList,
+      TaskItem,
+      Highlight,
+    ],
+    content: `
+      <p>This isn’t bold.</p>
+      <p><strong>This is bold.</strong></p>
+      <p><b>And this.</b></p>
+      <p style="font-weight: bold">This as well.</p>
+      <p style="font-weight: bolder">Oh, and this!</p>
+      <p style="font-weight: 500">Cool, isn’t it!?</p>
+      <p style="font-weight: 999">Up to font weight 999!!!</p>
+    `,
```

Comment:
❌ 
you hard coding the content, but it should come as a prop, you will also need onChange event